webportal/src/app/user/user-auth/user-auth.component.js

```diff
@@ -26,4 +26,8 @@ const checkToken = (callback) => {
   }
 };
 
-module.exports = { checkToken };
\ No newline at end of file
+const checkAdmin = () => {
+  return cookies.get('admin') === 'true';
```

Comment:
I’m wondering what if the response is hijacked and modified, or the webpage code is changed dynamically in Chrome’s F12 console?

Reply:
anyone can change anything in his browser, it doesn't have any effect to the back end.